### ISSUE TITLE
Pre-insall numpy and enable i2c/spi for rpi mcu for measuring resonances

### ISSUE THREAD

**Issue by guysoft** [guysoft/OctoKlipperPi#5]
Hey,
Considering shipping with numpy and i2c/spi enabled, so input shaping would be easier to set up.

Reference: https://www.klipper3d.org/Measuring_Resonances.html#bed-slinger-printers

**Comment by grlstuff** [comment]
Yes please, just resorted to a full manual install as i couldnt manage to get it to work with this version :-(
